### PR TITLE
Revisions to descriptions of cohort events

### PR DESCRIPTION
**Pull request by lamagnifica**
@catong these are the events that are emitted when things relating to cohorts happen. Basically, I wanted to get rid of "manually" in relation to course teams adding students to cohorts. But I took the opportunity to remove the Latin, inlcude xrefs to the B&R guide, and otherwise polish up these descriptions. No rush.
@stroilova, FYI

### CODE REVIEW

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 1]
```diff
@@ -2764,15 +2771,18 @@ student is assigned to one).
...
+*********************************
 
 When a user is added to a cohort group, the server emits an
-``edx.cohort.user_added`` event. A member of the course staff can add a user to
-a cohort group manually via the Instructor Dashboard (see
-:ref:`instructor_cohort_events`). The system automatically adds a user to the 
-default cohort group or a cohort group included in the course's 
-``auto_cohort_groups`` setting if the user accesses a discussion but has not 
-yet been assigned to a cohort group.
+``edx.cohort.user_added`` event. Users can be added to cohort groups by
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Preferred: focus on the actions by course team/system vs the users who are being added, in keeping with other paragraphs in this section.  "Members of the course team can add users to cohort groups. The system automatically adds..."

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 2]
```diff
@@ -3300,11 +3312,11 @@ event.
      - The display name of the cohort group.
 
 ``edx.cohort.user_add_requested``
-----------------------------------
+*********************************
 
-When an instructor or course staff member adds a student to a cohort group via
-the Instructor Dashboard, the server emits an ``edx.cohort.user_add_requested``
-event.
+When an instructor or course staff member adds a student to a cohort group on
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:3318]: Is it clear enough that this description includes adding students via a csv upload? If not too wordy/specific, perhaps add "either manually or using a csv file"?

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 3]
```diff
@@ -2764,15 +2771,18 @@ student is assigned to one).
      - The display name of the cohort group.
 
 ``edx.cohort.user_added``
-----------------------------------
+*********************************
 
 When a user is added to a cohort group, the server emits an
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:2776]: "student" rather than "user"? Seems that "student" is used in some places and "user" in others -- possible to choose one and be consistent?

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 4]
```diff
@@ -2764,15 +2771,18 @@ student is assigned to one).
...
-``edx.cohort.user_added`` event. A member of the course staff can add a user to
-a cohort group manually via the Instructor Dashboard (see
-:ref:`instructor_cohort_events`). The system automatically adds a user to the 
-default cohort group or a cohort group included in the course's 
-``auto_cohort_groups`` setting if the user accesses a discussion but has not 
-yet been assigned to a cohort group.
+``edx.cohort.user_added`` event. Users can be added to cohort groups by
+members of the course team. If a user who has not been assigned to a cohort
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: "student" rather than "user"?

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 5]
```diff
@@ -2764,15 +2771,18 @@ student is assigned to one).
...
+group accesses course content that is divided by cohort, the system
+automatically adds that user to a cohort group defined by the
+``auto_cohort_groups`` advanced setting or to the default cohort group.
+
+Additional events are emitted when members of the course team interact with
+the Instructor Dashboard to add a user to a group. See
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:2785]: "student" vs "user"

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 6]
```diff
@@ -2795,14 +2805,14 @@ yet been assigned to a cohort group.
      - The display name of the cohort group.
    * - ``user_id``
      - integer
-     - The numeric ID (from auth_user.id) of the added user.
+     - The numeric ID (from ``auth_user.id``) of the added user.
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:2809]: "student" vs "user"

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 7]
```diff
@@ -2795,14 +2805,14 @@ yet been assigned to a cohort group.
...
 
 ``edx.cohort.user_removed``
-----------------------------------
+*********************************
 
-When a user is removed from a cohort group (by being assigned to a different
-cohort group via the Instructor Dashboard), the server emits an
-``edx.cohort.user_removed`` event.
+When a course team member changes the cohort group assignment of a user on
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:2814]: "student" vs "user"?

**Comment by catong** [comment]
I did not comment on every occurrence, but it might be a good idea to make the use of "student" or "user" consistent throughout.

**Comment by catong** [comment]
:+1: with suggestions

**Comment by lamagnifica** [comment]
@catong I don't disagree about the user vs. student consistency issue. However, making that change throughout this chapter right now will make this branch really a brute to rebase with the separate branch for video events that I have going right now. I will open a DOC backlog task to make the wording consistent in the entire file.

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: It was that second sentence that I was trying to simplify. Reverting.

**Comment by lamagnifica** [comment]
@catong please ok

**Comment by catong** [comment]
:+1:

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 2 above)
Comment (stroilova) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:3318]: Thank you @lamagnifica -- did we end up specifying explicitly both scenarios:
- instructors manually adding students to a cohort (I'm assuming using a UI in the Instructor Dashboard or Studio?) , and
- instructors uploading a CSV of students with cohort assignments

By the way, do the events distinguish these two scenarios, and should we run the final language by the implementing team?

FYI : @mulby  , @gwprice ( ? ) , @cahrens ( ? )

Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:3318]: @stroilova , I just got these comments (several days after the PR was closed and merged and docs published). 
For your doc question, yes, both paths to adding students to a specific cohort are explicitly called out in teh documentation (lines 3320-3322 immediately below, or http://edx.readthedocs.org/projects/devdata/en/latest/internal_data_formats/tracking_logs.html#edx-cohort-user-added).
I do not believe that the T&L team changed the eventing in any way when they expanded the feature to allow course teams to semi-automate assignment of students to specific cohorts by uploading a CSV. Please open a JIRA ticket for the discovery or potential addition to the events. This PR is closed now, which was solely to remove "manually" etc. as stated in the description.
